fontbakery-check-ttf.py

```diff
@@ -1531,28 +1531,35 @@ def font_has_char(font, c):
       #-----------------------------------------------------
       logging.debug("METADATA.pb: Fontfamily is listed in Google Font Directory ?")
       url = 'http://fonts.googleapis.com/css?family=%s' % family.name.replace(' ', '+')
-      fp = requests.get(url)
-      if fp.status_code != 200:
-        logging.error('No family found in GWF in %s' % url)
-      else:
-        logging.info('OK: Font is properly listed in Google Font Directory.')
+      try:
+        fp = requests.get(url)
+        if fp.status_code != 200:
+          logging.error('No family found in GWF in %s' % url)
+        else:
+          logging.info('OK: Font is properly listed in Google Font Directory.')
+      except:
+        logging.warning("Failed to fetch Microsoft's vendorID list.")
```

Comment:
this message is wrong. Should be something like "Failed to query GWF at {}".format(url)

I'll cherry-pick this and fix this way.